.changeset/green-dolphins-cover.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-scaffolder-backend-module-bitbucket': minor
```

Comment:
Think this one should stay as patch as this isn't breaking and the package is 0.x

Reply:
updated

Reply:
@NeuralEvolution sorry - only the `@backstage/plugin-scaffolder-backend-module-bitbucket` should be patch. The `@backstage/plugin-scaffolder-backend` can be minor :)

Reply:
@benjdlambert updated